### ISSUE TITLE
Missing accessibility metadata reported as EPUB violation by Ace, ignored by EpubCheck

### ISSUE THREAD

**Issue by bitsgalore** [daisy/ace#211]
I just ran Ace on [this EPUB 2.0.1 file](https://www.dbnl.org/tekst/_hol003190301_01/ebook/_hol003190301_01.epub). It reports 4 `metadata-schema-accessmodesufficient` violations from the EPUB ruleset (caused by missing accessibility metadata fields in the .opf). However, EpubCheck 4.0.2 does not report these violations. This makes me wonder whether Ace is too strict on this, or perhaps EpubCheck is not thorough enough?

(On a side note, EpubCheck also reports 2 additional NCX-related EPUB violations (and serious ones at that!) for this EPUB that are not picked up by Ace, but [seeing that Ace currently ignores the NCX](https://github.com/daisy/ace/issues/21) this is expected.)

(Using Ace version 1.0.2)

**Comment by rdeltour** [comment]
Ace and EpubCheck are two different things, which scopes only overlap. EpubCheck is a strict conformance checker for the EPUB specifications, while Ace only looks at accessibility criteria.

On the one hand, conformance to the the [EPUB Accessibility 1.0 specification](http://www.idpf.org/epub/a11y/accessibility.html#sec-discovery) is not mandated by the EPUB 2.0.1 specification, so EpubCheck is correct in not reporting the missing metadata as a spec violation.

On the other hand, Ace (partially) evaluates conformance to the same EPUB Accessibility specification, so it is correct in reporting missing metadata as violations to the "Discovery" part of this specification.

In other words, it is not unusual nor unexpected that Ace report violations that are not raised by EpubCheck and vice-versa 😄.

Closing this issue as `invalid`, feel free to continue the discussion if you have any comments or questions!

**Comment by bitsgalore** [comment]
Thanks, I know that Ace Ace and Epubcheck have different scopes, but what confused me here is that those violations are reported under a ruleset that is called "EPUB" (which suggests it refers to the EPUB 3.x/2.x specs), whereas in reality it refers to the "EPUB Accessibility" spec (which, as you rightly point out, is something else altogether). So for the sake of clarity it might be helpful to explicitly mention "EPUB Accessibility" in the output report, rather than just "EPUB".

**Comment by rdeltour** [comment]
> So for the sake of clarity it might be helpful to explicitly mention "EPUB Accessibility" in the output report, rather than just "EPUB".

OK, I understand the confusion. We can definitely try to clarify that, thanks!
